machine/alphanetworks/alphanetworks_snx60a0_486t/installer.conf

```diff
@@ -0,0 +1,14 @@
+# Alpha Networks SNX60A0-486T ONIE installer configuration
+
+description="AlphaNetworks, SNX60A0-486T"
+
+# Default ONIE block device
+install_device_platform()
+{
+    echo /dev/sda
```

Comment:
Yes, It is always true. The device node, "/dev/sda", is always represented for the SSD in our device even the USB disk is plugged in.